Description: Use local intersphinx inventories.
 Fetching the inventories off the Internet made the build unreliable.
 This patch turns off the intersphinx feature for notmuch and urwid since
 they don't have a -doc sphinx package yet. This patch only makes sense in
 the Debian context.
Author: Simon Chopin <chopin.simon@gmail.com>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -256,7 +256,7 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
-    'http://docs.python.org/': None,
-    'http://notmuch.readthedocs.org/en/latest/': None,
-    'http://urwid.readthedocs.org/en/latest/': None
+    'file:///usr/share/doc/python-doc/html/': None,
+    #'http://notmuch.readthedocs.org/en/latest/': None,
+    #'http://urwid.readthedocs.org/en/latest/': None
 }
